docs/_docs/user-docs/recipe-groups.md

```diff
@@ -0,0 +1,150 @@
+For example, we might have:
+
+```
+FORMAT dicom
+
+%header
+
+ADD PatientIdentityRemoved Yes
+BLANK OrdValue
+KEEP Modality
+REPLACE id var:entity_id
+JITTER StudyDate var:entity_timestamp
+REMOVE ReferringPhysicianName
+```
+
+But what if we want to optimize our parsing by creating custom groups of tags
+that are based on the field names, or the values? This is the intended use
+case for groups - a group is a group of tags, either identified by
+fields or values, for which an action can be applied. For the examples
+below, we will use this sample header provided by [@wetzelj](https://github.com/wetzelj). Thank you!
+
+```
+(0008,0050) : SH   Len: 10     AccessionNumber                Value: [999999999 ]
+(0008,0070) : LO   Len: 8      Manufacturer                   Value: [SIEMENS ]
+(0008,1090) : LO   Len: 22     ManufacturersModelName         Value: [SOMATOM Definition AS+]
+(0009,0010) : LO   Len: 20     PrivateCreator10xx             Value: [SIEMENS CT VA1 DUMMY]
+(0010,0010) : PN   Len: 14     PatientsName                   Value: [SIMPSON^HOMER^J^]
+(0010,0020) : LO   Len: 12     PatientID                      Value: [000991991991 ]
+(0010,1000) : LO   Len: 8      OtherPatientIDs                Value: [E123456]
+(0010,1001) : PN   Len: 8      OtherPatientNames              Value: [E123456]
+(0010,21B0) : LT   Len: 90     AdditionalPatientHistory       Value: [MR SIMPSON LIKES DUFF BEER]
+(0019,1091) : DS   Len: 6      <Unknown Tag>                  Value: [E123456]
+(0019,1092) : DS   Len: 6      <Unknown Tag>                  Value: [M123456]
+```
+
+## Fields
+
+A fields section looks like the following:
+
+```
+FORMAT dicom
+
+%fields patient_info
+FIELD PatientID
+FIELD startswith:OtherPatient
+FIELD endswith:Name
+```
+
+There would be multiple ways to do this (for example you could have used `startswith:Patient` to target both `PatientsName`
+and `PatientID`) but generally this will produce a list of fields that are named "patient_info." Here is the list
+rendered out pretty:
+
+```
+patient_info
+------------
+PatientID
+OtherPatientIDs
+OtherPatientNames
+PatientsName
+```
+
+We can then use this in recipe header sections where we want to apply an action to one or more fields
+as follows:
+
+```
+%header
+
+REPLACE fields:patient_info func:generate_uid
+```
+
+And this reads nicely as "Replace fields defined in patient_info to be the variable 
+I'm defining with the function generate_uid (which should be added to each item 
+after lookup).
+
+This of course means that the actions supported for the `%fields` section includes:
+
+ - **FIELD** reference to a full name of a field, or any parsing of any [expander]({{ site.baseurl }}/examples/header-expanders/).
+
+
+## Values
+
+It could be that you want to generate a list of _values_ extracted from the dicom
+to use as flags for checking other fields. For example, if I know that the Patient's ID
+is in PatiendID, I would want to extract the patient's name from that field,
```

Comment:
Typo: 
> is in **PatiendID**, I would want to extract the patient's name from that field

Reply:
I wish I could say that I had sausage fingers, but I think they are more spiderlike. So I'll just say I had bad fingers here :)